backend/src/test/java/com/woowacourse/kkogkkog/acceptance/AuthAcceptanceTest.java

```diff
@@ -24,16 +24,42 @@ public class AuthAcceptanceTest extends AcceptanceTest {
         로그인에_성공한다(tokenRequest);
     }
 
+    @Test
+    void 이메일이_입력되지_않으면_로그인_할_수_없다() {
+        MemberAcceptanceTest.회원_가입에_성공한다(new MemberCreateRequest("rookie@gmail.com",
+                "password1234!", "rookie"));
+        TokenRequest tokenRequest = new TokenRequest("", "password1234!");
+
+        ExtractableResponse<Response> response = 로그인을_요청한다(tokenRequest);
+
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.BAD_REQUEST.value());
+    }
```

Comment:
인수테스트에서 입력값에 대한 테스트를 다루는 것보다는 현재 ControllerTest를 DocumentationTest로 변경하고 입력값에 대한 테스트를 ControllerTest로 만들면 어떨까 싶네요. 인수테스트에서 다루기에는 너무 세부적인 사항이라고 생각합니다.

공백에 대한 검증을 할때 null 과 empty를 모두 검증해주는 것을 선호해서 이런식으로 작성했었습니다.

```suggestion
   @ParameterizedTest
   @NullAndEmptySource
    void 이메일이_입력되지_않으면_로그인_할_수_없다(String email) {
        MemberAcceptanceTest.회원_가입에_성공한다(new MemberCreateRequest("rookie@gmail.com",
                "password1234!", "rookie"));
        TokenRequest tokenRequest = new TokenRequest(email, "password1234!");

        ExtractableResponse<Response> response = 로그인을_요청한다(tokenRequest);

        assertThat(response.statusCode()).isEqualTo(HttpStatus.BAD_REQUEST.value());
    }
```

Reply:
좋은 방법이네요.